package corejava;

public class Try_catch {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
int d=0;
int n=20;
try {
int fraction = n/d;
System.out.println("End of Main");
}	
	catch (ArithmeticException e) {
		System.out.println("you can not divide by zero");
	}	
		
		
		
	}

}
